secret.py

```
@@ -0,0 +1,14 @@
+def log_in(password):
+    conn = sqlite3.connect('users.db')
+    cursor = conn.cursor()
+
+    hashed_password = hash_password(password)
```

Ruff (0.8.2)

Undefined name `hash_password`

(F821)